tests/test_scxml.py

```diff
@@ -0,0 +1,33 @@
+import xml.etree.ElementTree as ET
+from typing import Optional, Dict, List
+import json
+import pytest
+from xstate.scxml import scxml_to_machine
+
+test_dir = "node_modules/@scion-scxml/test-framework/test"
+
+test_groups: Dict[str, List[str]] = {"actionSend": ["send1"]}
+
+test_files = [
```

Comment:
For each test, e.g., `actionSend/send1`, we get two files:

1. `actionSend/send1.scxml` - this is the SCXML file that defines the machine
2. `actionSend/send1.json` - this is the test file that contains a list of events to be executed and the expected "configuration" of the machine when that event is received.

A machine configuration is what states the machine are in (for example, a traffic light machine can be in the configuration of `"red"` and `"red.walking"`, because those two state nodes are active)